packages/instrumenter/src/mutation-level/mutation-level.ts

```diff
@@ -47,9 +48,30 @@ export interface MutationLevel {
   StringLiteral?: StringLiteral[];
   UnaryOperator?: UnaryOperator[];
   UpdateOperator?: UpdateOperator[];
-  [k: string]: unknown;
+  [k: string]: MutatorDefinition[] | string | undefined;
```

Comment:
I don't think I quite understand this syntax, what does this mean?

Reply:
It's an index signature for the object. Without it, you wouldn't be able to to stuff like
`
const string = 'StringLiteral';
level[string];
`

I changed it so I would avoid some type castings which were necessary with `unknown`